packages/contentstack-bootstrap/src/bootstrap/utils.ts

```diff
@@ -123,11 +123,13 @@ const envFileHandler = async (
   let filePath;
   let fileName;
   let customHost;
+  let previewHost:string;
   const regionName = region && region.name && region.name.toLowerCase();
-  const managementAPIHost = region.cma && region.cma.substring('8');
+  previewHost = region.cda.replace('cdn','rest-preview');
```

Comment:
@cs-raj , Can we use `const previewHost` directly on line 128 and also optional chaining in case of `region.cda?.replace()`;

Reply:
added